backend/db/migrate/20230608080932_rename_api_entreprise_scopes_in_enrollments.rb

```diff
@@ -0,0 +1,127 @@
+class RenameApiEntrepriseScopesInEnrollments < ActiveRecord::Migration[7.0]
+  def up
+    valid_enrollments.find_each do |enrollment|
+      scopes = enrollment.scopes.map do |scope|
+        case scope
+        when "associations"
+          "open_data"
+        when "associations_donnees_protegees"
+          "associations_djepva"
+        when "attestations_agefiph"
+          "attestations_agefiph"
+        when "attestations_fiscales"
+          "attestation_fiscale_dgfip"
+        when "attestations_sociales"
+          "attestation_sociale_urssaf"
+        when "bilans_entreprise_bdf"
+          "bilans_bdf"
+        when "fntp_carte_pro"
+          "open_data"
+        when "certificat_cnetp"
+          "certification_cnetp"
+        when "msa_cotisations"
+          "cotisations_msa"
+        when "certificat_opqibi"
+          "certification_opqibi"
```

Comment:
La certification OPQIBI étant en open data, on la met dans le pack de droits open data

```suggestion
          "open_data"
```